extension/sqlsmith/statement_generator.cpp

```diff
@@ -73,13 +88,29 @@ std::shared_ptr<GeneratorContext> StatementGenerator::GetDatabaseState(ClientCon
 			result->tables_and_views.push_back(entry);
 		});
 	}
+	if (context.transaction.HasActiveTransaction()) {
+		context.transaction.Commit();
+	}
 	return result;
 }
 
 unique_ptr<SQLStatement> StatementGenerator::GenerateStatement() {
 	if (RandomPercentage(80)) {
 		return GenerateStatement(StatementType::SELECT_STATEMENT);
 	}
+	if (RandomPercentage(40)) {
+		if (RandomPercentage(20)) {
```

Comment:
haha this can be `RandomPercentage(50)`

That way, if the code falls in here, then there is a 50% chance we generate attach and use

Reply:
A good point! Attach and use deserve to be generated together more often